### DISCUSSION TITLE
Sailing Mode expectations

### DISCUSSION THREAD

**Discussion by door665** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1157] [Q&A]
Hi all! 

So, I use my MBP in clamshell mode 95% of the time. I've started using Al Dente 2 days ago, yesterday did a full calibration and have the charging limit set to 75% and Sailing Mode to 5%.

As I was working today I noticed it went from 75% to 71%. Is it normal for this amount of discharge to happen?! I'm a bit worried about it. This MBP is brand new, have had it for 2 weeks.

Thanks!

**Comment by MatthiasKerbl** [comment]
Hi @door665,

It depends on your setup if your scenario is unusual.
Which power adapter and cable do you use to charge your MacBook?

Best,
Matthias

**Reply by door665** [reply]
I'm using Apple's own charger, the 140W charger (M3 Max). It hasn't happened again, might have been something I don't know.

I know this is a bit off-topic but there's tons of weird information on reddit and even macrumors forums about Al Dente, people complaining it killed their battery, or that their battery's health went down too fast after installing it compared to before etc. As far as I understand, all Al Dente is doing is keeping the max charge % at whatever you set and that's it right? Apart from other features (that are super useful) like Sailing mode and stop charging when sleeping etc. 

I posted about weird sleep / darkwake pmset logs on Apple's forums and just look at this response...: https://discussions.apple.com/thread/255437612?cid=em-com-apple_watches_email_thread_owner-view_the_full_discussion-en-us-11282023&sortBy=best

**Reply by door665** [reply]
Also, @MatthiasKerbl do you know what the expected drop off of 5% usually is? When in clamshell perpetually plugged in? Using the 140w charger should mean that the battery should not reach said 5% sail for weeks? Or am I completely off the mark?

Just measuring expectations and making sure nothing is "wrong" :)

**Reply by MatthiasKerbl** [reply]
Hi @door665,

A drop of 4% within two days in your setup is a little bit unusual but not concerning especially if it didn't happen since.
For your setup it might take weeks before the battery drops 5% during Sailing Mode.

We are aware of some of the concerns people have about AlDente on Reddit and other Forums. Unfortunately, we don't have the resources to go through all these Reddit and other forum posts. We can only offer proper help to paying customers through our official support. However, we will try to improve our communications to maybe mitigate some of the concerns.
In my opinion, most of these concerns are due to misunderstandings about what AlDente does, what it can do, what it can not do, and how all of this is related to battery health especially also regarding miscalibration of batteries.
The features of AlDente are based on battery research on how to increase battery lifespan.

Best,
Matthias
